### ISSUE TITLE
Pro Menu breakpoint-specific background color doesn't work

### ISSUE THREAD

**Issue by swinggraphics** [soflyy/oxygen-bugs-and-features#1640]
**Describe the bug**
If you set the background color for the mobile menu in All Devices mode and change it while a smaller breakpoint is selected, the color does not change in the editor. It will be correct on reload and on the frontend.

**Have you tried all the steps at https://oxygenbuilder.com/documentation/troubleshooting/troubleshooting-guide/?**
*Yes*

**Comment by KittenCodes** [comment]
@swinggraphics Please could you record a quick video showing the steps you're taking: https://www.loom.com/? I've not been able to replicate this.

**Comment by swinggraphics** [comment]
Certainly. https://www.dropbox.com/s/hc81r2ki4nvr2lp/Oxygen%20Pro%20Menu%20bg%20color.mov?dl=0

**Comment by KittenCodes** [comment]
> Certainly. https://www.dropbox.com/s/hc81r2ki4nvr2lp/Oxygen%20Pro%20Menu%20bg%20color.mov?dl=0

Great, thanks! I've reported this internally.
